examples/standalone_mode_demo.ipynb

```diff
@@ -0,0 +1,609 @@
+    "    eval_metric=config.eval_metric,\n",
+    "    multilabel=config.multilabel)\n",
+    "trainer.setup_evaluator(evaluator)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/envs/pytorch/lib/python3.10/site-packages/torch/nn/parallel/distributed.py:1915: UserWarning: You passed find_unused_parameters=true to DistributedDataParallel, `_set_static_graph` will detect unused parameters automatically, so you do not need to set find_unused_parameters=true, just be sure these unused parameters will not change during training loop while calling `_set_static_graph`.\n",
+      "  warnings.warn(\n",
+      "INFO:root:Part 0 | Epoch 00000 | Batch 000 | Loss: 5.0432 | Time: 1.2219\n",
+      "INFO:torch.nn.parallel.distributed:Reducer buckets have been rebuilt in this iteration.\n",
+      "INFO:root:Epoch 0 take 1.325 seconds\n",
+      "INFO:root:successfully save the model to nc_model/epoch-0\n",
+      "INFO:root:Time on save model: 0.007 seconds\n",
+      "WARNING:root:We do not export the state of sparse optimizer\n",
+      "INFO:root:Part 0 | Epoch 00001 | Batch 000 | Loss: 3.1350 | Time: 0.0110\n",
+      "INFO:root:Epoch 1 take 0.110 seconds\n",
+      "INFO:root:successfully save the model to nc_model/epoch-1\n",
+      "INFO:root:Time on save model: 0.006 seconds\n",
+      "WARNING:root:We do not export the state of sparse optimizer\n",
+      "INFO:root:Part 0 | Epoch 00002 | Batch 000 | Loss: 2.0955 | Time: 0.0113\n",
+      "INFO:root:Epoch 2 take 0.111 seconds\n",
+      "INFO:root:successfully save the model to nc_model/epoch-2\n",
+      "INFO:root:Time on save model: 0.004 seconds\n",
+      "WARNING:root:We do not export the state of sparse optimizer\n",
+      "INFO:root:Part 0 | Epoch 00003 | Batch 000 | Loss: 1.6824 | Time: 0.0108\n",
+      "INFO:root:Epoch 3 take 0.107 seconds\n",
+      "INFO:root:Part 0 | Epoch 00004 | Batch 000 | Loss: 1.4751 | Time: 0.0097\n",
+      "INFO:root:Epoch 4 take 0.106 seconds\n",
+      "INFO:root:Part 0 | Epoch 00005 | Batch 000 | Loss: 1.1333 | Time: 0.0097\n",
+      "INFO:root:Epoch 5 take 0.117 seconds\n",
+      "INFO:root:Part 0 | Epoch 00006 | Batch 000 | Loss: 1.0918 | Time: 0.0097\n",
+      "INFO:root:Epoch 6 take 0.116 seconds\n",
+      "INFO:root:Part 0 | Epoch 00007 | Batch 000 | Loss: 0.8312 | Time: 0.0104\n",
+      "INFO:root:Epoch 7 take 0.110 seconds\n",
+      "INFO:root:Part 0 | Epoch 00008 | Batch 000 | Loss: 0.7752 | Time: 0.0103\n",
+      "INFO:root:Epoch 8 take 0.110 seconds\n",
+      "INFO:root:Part 0 | Epoch 00009 | Batch 000 | Loss: 0.4196 | Time: 0.0102\n",
+      "INFO:root:Epoch 9 take 0.109 seconds\n",
+      "INFO:root:Peak GPU Mem alloc: 20.3286 MB\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Train the model with the trainer\n",
+    "trainer.fit(\n",
```

Comment:
If you can set up a task_tracker for the trainer, you will see the evaluation and testing results during the training epochs. Then no need to call the `trainer.evaluator._history` internal attribute. We didn't expose this attribute.

Reply:
task_tracker is not exposed API. Also, setting up task_tracker requires the `GSConfig` again. We can use task_tracker   if initializing doesn't require GSConfig.